docs/source/fitting/index.rst

```diff
@@ -6,7 +6,7 @@ Fitting Overview
 
 .. raw :: html
 
-    <blockquote>This is technical documentation relating to fitting capabilities in Mantid. The fitting 
+    <intro>This is technical documentation relating to fitting capabilities in Mantid. The fitting 
```

Comment:
I'd rather not use non standard element names.

This should either be a style that can be applied to any element as required, e.g.:
```css
.smaller_text {
  font-size: 80%;
}
```

Or (possibly the better solution) if the global font sizes are genuinely an issue then they should be changed everywhere.

Reply:
I'm not sure that global font size is an issue - or rather there are so many different lengths of list and description that there is no one-size-fits all solution. Now, that could be part of a larger issue in that we should not really have so many different scenarios, but that in turn requires major reworking of the docs structure, which is beyond this PR.